refnx/reflect/test/simulation.dat

```diff
@@ -0,0 +1,51 @@
+0.051793 0.00034985 5.0354e-06
```

Comment:
`mdsimulation.dat`

Reply:
Done. and changed all the other files appropriately.